packages/squiggle-lang/src/rescript/FunctionRegistry/Library/FR_List.res

```diff
@@ -81,25 +73,18 @@ module Internals = {
     accessors: ProjectAccessorsT.t,
     reducer: ProjectReducerFnT.t,
   ) => {
-    let rMappedList = aValueArray->Belt.Array.reduceReverse(Ok(list{}), (rAcc, elem) =>
-      rAcc->E.R.bind(_, acc => {
-        let rNewElem = Reducer_Expression_Lambda.doLambdaCall(
-          aLambdaValue,
-          list{elem},
-          accessors,
-          reducer,
-        )
-        rNewElem->E.R2.fmap(newElem => {
-          switch newElem {
-          | IEvBool(true) => list{elem, ...acc}
-          | _ => acc
-          }
-        })
-      })
-    )
-    let result =
-      rMappedList->E.R2.fmap(mappedList => mappedList->Belt.List.toArray->Wrappers.evArray)
-    result
+    Js.Array2.filter(aValueArray, elem => {
```

Comment:
Any reason you didn't use E.A.filter? If the implementation there is suboptimal, you can just change it there.

Reply:
I wrote this before I asked that question on #squiggle-dev, and didn't got around to updating the PR.